saleor/site/migrations/0035_sitesettings_separate_state_remove_order_settings.py

```diff
@@ -0,0 +1,65 @@
+# Generated by Django 3.2.16 on 2022-12-14 09:28
+
+from django.db import migrations, models
+
+# Separate state and database changes in migration to avoid downtime.
+# For next release will be prepared migration to remove the field from database.
+
+# Database fields in this migration are changed to null to allow adding new rows to table.
+# To reverse that operation, we will set all values for `SiteSettings` regarding the
+# first channel in alphabetical order.
+
+
+def update_order_settings_values_to_match_first_channel(apps, schema_editor):
+    SiteSettings = apps.get_model("site", "SiteSettings")
+    Channel = apps.get_model("channel", "Channel")
+
+    channel = Channel.objects.order_by("slug").first()
```

Comment:
Still not sure that we should use this channel as default.
Maybe better to fetch the first active channel ordered by `pk`?

Reply:
@fowczarek Yeah, agree with `isActive`. We should add them to all deprecated resolvers also (we use there a first channel also). 
It is filtered by `slug` to keep the same way of picking the channel as we do for deprecated SiteSettings resolvers.

Reply:
I think we shouldn't base it on a dynamic field because the first query could return `Channel(slug="ab")` and after a few seconds, the first channel returned by this query could be `Channel(slug="aa")`. I think we should always base on pk in this kind of situation.